docs/tutorials/backends/t2star_backend.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+
+from qiskit.utils import apply_prefix
+from qiskit.providers import BaseBackend
+from qiskit.providers.models import QasmBackendConfiguration
+from qiskit.result import Result
+from qiskit.test import QiskitTestCase
+from qiskit_experiments.composite import ParallelExperiment
+from qiskit_experiments.characterization import T2StarExperiment
+
+
+# Fix seed for simulations
+SEED = 9000
```

Comment:
Is the seed to be fixed here or in the test file?

Reply:
I think it should be fixed in both.